README.md

```diff
@@ -12,3 +12,63 @@ The _RFC - Py-Spiffe features and use cases_ document is public at: https://docs
 * Gomez Coronel Andres (HPE)
 * Lambrecht Max (HPE)
 
+
+## Development setup
+
+### Prerequisites
+For basic development you will need:
+
+* Python 3.6
+* Pipenv (https://github.com/pypa/pipenv)
+* Pyenv (https://github.com/pyenv/pyenv)
+
+### Setup the environment
+
+Use `pyenv` to install the different version of Python.
+Python 3.6 is required for development and the other versions are required for testing.
+```
+pyenv install 3.6.x
+pyenv install 3.7.x
+pyenv install 3.8.x
+pyenv install 3.9.x
+```
+
+Clone the repository
+```
+git clone git@github.com:HewlettPackard/py-spiffe.git
+```
+
+cd to the py-spiffe directory
+```
+cd py-spiffe
+```
+
+Use the installed versions to define the specific versions for development and testing
+```
+pyenv local 3.6.12 3.7.0 3.8.0 3.9.0
```

Comment:
3.7.9 and 3.8.6 are the lastest patch versions

Reply:
Those versions are only to show as example. Actually the install step does not specify any particular patch.